index.html

```diff
@@ -0,0 +1,122 @@
+
+<a href="#declaring_variables" class="nav-link">Declaring variables</a><br><br><hr>
+
+<a href="#declaring_variables" class="nav-link">Declaring variables</a><br><br><hr>
+
+<a href="#declaring_variables" class="nav-link">Declaring variables</a><br><br><hr> -->
+
+        </nav>
+    </div>
+
+<!-------- Right Column--------->
+    <div class="col2">
+        <main id="main-doc">
+            <section id="introduction" class="main-section">
+                <header>INTRODUCTION</header>
+                <p>JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language. Inside a host environment (for example, a web browser), JavaScript can be connected to the objects of its environment to provide programmatic control over them.
+                <p>JavaScript contains a standard library of objects, such as Array, Date, and Math, and a core set of language elements such as operators, control structures, and statements. Core JavaScript can be extended for a variety of purposes by supplementing it with additional objects; for example:
+                <ul>
+                    <li>Client-side JavaScript extends the core language by supplying objects to control a browser and its Document Object Model (DOM). For example, client-side extensions allow an application to place elements on an HTML form and respond to user events such as mouse clicks, form input, and page navigation.</li>
+                    <li>Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on a server. For example, server-side extensions allow an application to communicate with a database, provide continuity of information from one invocation to another of the application, or perform file manipulations on a server.</li>
+                </ul>
+            </section>
+
+            <section id="what_you_should_already_know" class="main-section">
+                <header>What you should already know</header>
+                <p>This guide assumes you have the following basic background:
+                    <ul>
+                        <li>A general understanding of the Internet and the World Wide Web (WWW).</li>
+                        <li>Good working knowledge of HyperText Markup Language (HTML).</li>
+                        <li>Some programming experience. If you are new to programming, try one of the tutorials linked on the main page about JavaScript.</li>
+                    </ul>
+
+            </section>
+
+<section id="javascript_and_java" class="main-section">
+<header>Javascript and Java</header>
+<p> JavaScript and Java are similar in some ways but fundamentally different in some others. The JavaScript language resembles Java but does not have Java's static typing and strong type checking. JavaScript follows most Java expression syntax, naming conventions and basic control-flow constructs which was the reason why it was renamed from LiveScript to JavaScript.
```

Comment:
I would like to suggest that the header tags here be replaced with h3 or h4 (but not h1)